sites/friday/src/projects/change-project-timezone.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Change the project timezone
+description: See how to change the timezone for a project and what it affects.
+---
+
+The project timezone affects [automated backups](../environments/backup.md).
+
+The project timezone doesn't affect:
+
+- [App runtime](../create-apps/timezone.md).
+- [Cron jobs](../create-apps/app-reference.md#crons).
+- [System logs](../increase-observability/logs/_index.md). UTC is the default timezone for all logs.
+
+To change the timezone for a project, follow these steps:
+
+1. Open the project where you want to change the timezone.
+2. Click {{< icon settings >}} **Settings**.
+3. Select the timezone from the list.
+4. Click **Save**.
```

Comment:
maybe we need to add a link (to this page) into the `security/backups.html` page, and add a note on that backup page to let the reader know that the schedule of backups is based on the project timezone?